frontend/src/types/AdminDashboardTypes.ts

```diff
@@ -1,12 +1,18 @@
-export interface ModulePreviewProps {
+export interface Module {
+  moduleId: string;
   title: string;
   published: boolean;
   imageLink: string;
 }
 
+export interface ModulePreviewProps extends Module {
+  courseId: string;
+}
+
```

Comment:
kept this separate since I don't expect `courseId` to exist on the GraphQL module response object?